test/system/systemTestResult.js

```diff
@@ -0,0 +1,22 @@
+const Promish = require('promish');
+
+const testSystem = (traders) => {
+  const promises = traders.map(trader => {
+    return trader.showBalanceDetail(trader.config)
```

Comment:
trader.showBalanceDetail doesn't actually fetch the balances, we should fetch them via the client before running checks